Aptoide/aptoide-client-v8
# [APP-1003] Created base aptoide network module

**What does this PR do?**

This PR aims at creating a base module with all base classes for network related objects. We were duplicating this in our branches and features without it being necessary.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] BaseV7Response.kt

**How should this be manually tested?**

Open the app and check that everything is well integrated on search. This should be integrated on other features after this PR is merged

**What are the relevant tickets?**

  Tickets related to this pull-request: [APP-1003](https://aptoide.atlassian.net/browse/APP-1003)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
